moveit_core/robot_model/include/moveit/robot_model/robot_model.h

```diff
@@ -239,6 +239,9 @@ class RobotModel
   /** \brief Get a link by its name. Output error and return NULL when the link is missing. */
   LinkModel* getLinkModel(const std::string& link);
 
+  /** \brief Get earliest link model that is rigidly connected to the given link */
```

Comment:
define "earliest" - is this the parent link? the root link? i guess with the term rigidly it means:

    joint1 - linkA - fixedJoint1 - linkB
    getRigidlyConnectedParentLinkModel(linkB)

return linkA correct?

Reply:
OK, obviously comment was too brief. I added more explanation.